source/guide_seafile.rst

```diff
@@ -192,15 +192,29 @@ Change seahub config; Edit ``~/seafile/conf/seahub_settings.py`` and  add the fo
 .. warning:: Replace ``isabell`` with your username!
 
 .. code-block:: console
-  :emphasize-lines: 1,2
-
+  
   SITE_BASE = 'https://isabell.uber.space'
   SITE_NAME = 'isabell.uber.space'
-
+  
   SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
-
+   
   FILE_SERVER_ROOT = SITE_BASE + '/seafhttp'
   CSRF_TRUSTED_ORIGINS = [SITE_NAME]
+  
+  #redirect to $USER tmp, avoid conflict with other users
+  CACHE_DIR = "/home/isabell/seafile/tmp/logs"
```

Comment:
```suggestion
  CACHE_DIR = os.path.expanduser("~/seafile/tmp")
```

1. Why `.../logs`? It seems like only caching data will be stored there, no logs.
2. It is common to use `os.path.expanduser()` in django configs to make them more portable. Be sure to import it, though.